Library/Homebrew/cmd/test-bot.rb

```diff
@@ -155,6 +156,14 @@ def run
           working_dir.cd { exec(*@command) }
         end
         write.close
+        if travis
+          print_thread = Thread.new do
+            while true do
+              printf "."
+              sleep 60
+            end
+          end
+        end
```

Comment:
What's the advantage to that? Seems a lot more complex. Having a e.g. one minute wait seems to make sense though (could just reverse the sleep and print). Creating/destroying a thread is easy enough to do.